test/built-ins/Atomics/add/bigint/non-shared-bufferdata.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2020 Rick Waldron. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-atomics.add
+description: >
+  Atomics.add will operate on TA when TA.buffer is not a SharedArrayBuffer
+includes: [testBigIntTypedArray.js]
+features: [ArrayBuffer, Atomics, BigInt, TypedArray]
+---*/
+testWithBigIntTypedArrayConstructors(TA => {
+  const buffer = new ArrayBuffer(BigInt64Array.BYTES_PER_ELEMENT * 4);
+  const view = new TA(buffer);
+  assert.sameValue(Atomics.add(i64a, 0, 1n), 0n, 'Atomics.add(i64a, 0, 1n) returns 0n');
```

Comment:
missing `i64a` reference

Reply:
Naw, it should be using `view`. I'm not sure how I messed that up on all of these test 🤦

Reply:
I've problems like this where I left files unsaved in the code editor and then committed in progress changes. My guess is that you probably did something similar.